plugins/shortcuts/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@backstage/plugin-shortcuts",
+  "version": "0.1.1",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/core": "^0.7.5",
+    "@backstage/theme": "^0.2.5",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.45",
```

Comment:
Don't actually see this package being referenced anywhere; is it needed?

Reply:
👍

Reply:
still in here!

Reply:
Had a problem removing this as the plugin template requires it: https://github.com/backstage/backstage/blob/master/packages/cli/templates/default-plugin/package.json.hbs#L31